src/System.IO.Compression/src/System/IO/Compression/ZLibNative.cs

```diff
@@ -322,7 +315,7 @@ public ErrorCode DeflateInit2_(CompressionLevel level, int windowBits, int memLe
                 try { }
                 finally
                 {
-                    errC = Interop.zlib.DeflateInit2_(ref _zStream, level, CompressionMethod.Deflated, windowBits, memLevel, strategy, ZLibVersion);
+                    errC = Interop.zlib.DeflateInit2_(ref _zStream, level, CompressionMethod.Deflated, windowBits, memLevel, strategy);
```

Comment:
We can remove the ```try { } finally { }``` and just leave the contents of the finally block, here and elsewhere this pattern is followed.